src/Pages/Infrastructure/Monitors/Components/Filters/index.jsx

```
@@ -0,0 +1,84 @@
+		const selectedValues = event.target.value;
+		setSelectedStatus(selectedValues.length > 0 ? selectedValues : undefined);
+
+		if (selectedValues.length === 0 || selectedValues.length === 2) {
+			setToFilterStatus(undefined);
+		} else {
+			setToFilterStatus(selectedValues[0] === "Up" ? "true" : "false");
+		}
+	};
+
+	const isFilterActive = useMemo(() => {
+		return selectedStatus?.length ?? 0 > 0;
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)